micropython/ssa/core.py

```
@@ -26,7 +24,7 @@ def __init__(self):
         self.BASE_ACTION_TOPIC: str
         self.REGISTRATION_TOPIC: str
 
-        self.__wlan: network.WLAN | None = None
+        self.__wlan: WLAN | None = None
         self.__mqtt: MQTTClient | None = None
 
         self.__action_cb_dict: Dict[str, funtion] = {}
```

Ruff (0.8.2)

Undefined name `Dict`

(F821)